### PR TITLE
remove hack based on runnumber=0

### PR DESCRIPTION
**Pull request by belforte**

### CODE REVIEW

File: src/python/TaskWorker/Actions/DBSDataDiscovery.py [hunk 1]
```diff
@@ -434,22 +434,11 @@ def executeInternal(self, *args, **kwargs):
             secondaryBlocksWithLocation = secondaryLocationsMap.copy().keys()
 
         # filter out TAPE locations
-        # temporary hack until we have a new config. parameter: allow user to accpet a partial dataset
-        # by inserting '0' as (first element of) runRange
-        usePartialDataset = False
-        runRange = kwargs['task']['tm_split_args']['runs']
```
Comment (novicecpp) [inline src/python/TaskWorker/Actions/DBSDataDiscovery.py:440]: Do not delete `runRange`. It is needed for [line 480](https://github.com/dmwm/CRABServer/blob/5bd657afce5387c95bfe830285fbda01cfc88e48/src/python/TaskWorker/Actions/DBSDataDiscovery.py#L480).

**Review by novicecpp — CHANGES_REQUESTED** [review]

**Comment by novicecpp** [comment]
Weird. I think I already reviewed this and you already merged it to master sometimes ago.
@belforte Did you really create PR today or it is reappear because something wrong with GitHub (GitHub server had some issues around lunch today)?

**Comment by belforte** [comment]
yes it is very strange. I also remember having done this already and was surprised to see old code still there. I guess I messed up in some PR based on stale branch :-(

**Review by novicecpp — APPROVED** [review]
LGTM

**Comment by belforte** [comment]
indeed there was already a reviewed PR for same topic. Which I had forgotten
Will now close https://github.com/dmwm/CRABServer/pull/7551

**Pull request merged**